src/main/java/io/github/bbortt/event/planner/repository/SectionRepository.java

```diff
@@ -23,4 +24,6 @@
 
     @Modifying
     long deleteAllByLocationId(Long locationId);
+
+    Optional<Section> findOneByNameAndLocationProjectId(String name, Long projectId);
```

Comment:
da steht schon byName**and**Location aber dort fehlt doch die locationId als parameter oder?

Reply:
es ist sql mässig ein `JOIN location JOIN project`, zweiteres wird über ersteres referenziert.